webapp/loggedin/appSideBar/appSideBarFooter.js

```diff
@@ -2,41 +2,36 @@ import React from 'react'
 
 import AppContext from '../../app/appContext'
 
-class AppSideBarFooter extends React.PureComponent {
-  render () {
-    const { logout, opened } = this.props
-    const { i18n } = this.context
+const AppSideBarFooter = ({ logout, opened }) => {
+  const { i18n } = React.useContext(AppContext)
 
-    return (
-      <div className="app-sidebar__footer">
-        <a className="btn btn-s btn-of-light-xs"
-           onClick={() => logout()}
-           style={{
-             display: 'flex',
-             alignItems: 'baseline',
-           }}>
-          <span className={`icon icon-exit ${opened ? ' icon-left' : ''}`}
-                style={{ transform: 'scaleX(-1)' }}/>
-          {
-            opened
-              ? <span>{i18n.t('sidebar.logout')}</span>
-              : null
-          }
-        </a>
+  return (
+    <div className="app-sidebar__footer">
+      <a className="btn btn-s btn-of-light-xs"
+         onClick={() => logout()}
+         style={{
+           display: 'flex',
+           alignItems: 'baseline',
+         }}>
+        <span className={`icon icon-exit ${opened ? ' icon-left' : ''}`}
+              style={{ transform: 'scaleX(-1)' }}/>
+        {
+          opened
+            ? <span>{i18n.t('sidebar.logout')}</span>
+            : null
+        }
+      </a>
 
-        <a className="btn btn-of-link app-sidebar__btn-of"
-           href="http://www.openforis.org"
-           target="_blank">
-          {
-            opened
-              ? i18n.t('sidebar.openForis') : i18n.t('sidebar.openForisShort')
-          }
-        </a>
-      </div>
-    )
-  }
+      <a className="btn btn-of-link app-sidebar__btn-of"
+         href="http://www.openforis.org"
+         target="_blank">
+        {
+          opened
+            ? i18n.t('sidebar.openForis') : i18n.t('sidebar.openForisShort')
```

Comment:
can the ternary operator "opened ? : " go inside i18n.t ? 
e.g. i18n.t(opened ? 'sidebar.openForis' : 'sidebar.openForisShort') or even i18n.t(`sidebar.openForis${opened ? 'Short' : ''}`)

Reply:
yes, moving it inside i18n